apps/web/pages/v2/apps/installed/[category].tsx

```diff
@@ -202,3 +198,25 @@ export default function InstalledApps() {
     </InstalledAppsLayout>
   );
 }
+
+export const getStaticProps: GetStaticProps = (ctx) => {
```

Comment:
Works well, but don't fully understand why it's required and why it wasn't working earlier.